raiden/src/raiden.ts

```diff
@@ -168,7 +159,19 @@ export class Raiden {
                         )
                       : Zero,
                   ),
-                }),
+                  capacity = channel.own.deposit.add(balance);
+                return (partner2raidenChannel[partner] = {
```

Comment:
Might be difficult to understand for contributors, could be prevented by eslint's `no-return-assign`

Reply:
Hmm, I may split it in a new var, but it'd be pointless as it'd be used only once on the return below. I'll check if `transform` requires the return to be in place.